auth/src/main/java/com/firebase/ui/auth/ErrorCodes.java

```diff
@@ -0,0 +1,20 @@
+package com.firebase.ui.auth;
+
+/**
+ * Error codes retrieved from {@link IdpResponse#getErrorCode()}.
+ */
+public final class ErrorCodes {
+    private ErrorCodes() {
+        // We don't want people to initialize this class
+    }
+
+    /**
+     * Sign in failed due to lack of network connection
+     **/
+    public static final int NO_NETWORK = 10;
```

Comment:
No, I don't think so. It should be an error code, but to keep backwards compatibility I'm [returning it in `SignInDelegate`](https://github.com/firebase/FirebaseUI-Android/pull/426/files#diff-741502abb522ecd56e7c30231e015bdbR83) which the [old `ResultCodes` points to](https://github.com/firebase/FirebaseUI-Android/pull/426/files#diff-314d061f90014b41124025139571c0e3R18).